helm/templates/deployment.yaml

```diff
@@ -58,6 +58,11 @@ spec:
               path: /liveness
               port: http
           {{- if .Values.resources.enabled }}
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.initialDelaySeconds }}
+            httpGet:
+              path: /liveness
+              port: http
```

Comment:
misplaced, no condition should be set fo readiness probe

Reply:
done